docs/topics/proc_configuring-your-application-to-use-vertx.adoc

```diff
@@ -22,7 +22,7 @@ Specify the `<type>pom</type>` and `<scope>import</scope>`:
       <dependency>
         <groupId>io.vertx</groupId>
         <artifactId>vertx-dependencies</artifactId>
-        <version>${vertx.version}</version>
+        <version>{version-vertx}</version>
```

Comment:
@inoxx03 I was pointing mainly to this line in my comment - this change is not really necessary :)

Reply:
Oh and btw you mentioned different guy previously :D